.gitignore

```diff
@@ -3,3 +3,4 @@
 /.vagrant
 /terraform.tfstate*
 /dist
+site.retry
```

Comment:
I think we should leave this out

Reply:
Just wondering, because it's not really relevant to the PR (fair point)? Otherwise it seems similar to `**/*-console.log` - an artifact you wouldn't really want to get checked in

Reply:
Sorry, ignore that comment completely I processed gitignore as un-ignore.